startupheroes-checks/src/main/resources/es/startuphero/checkstyle/startupheroes_checks.xml

```diff
@@ -191,6 +191,13 @@
     <module name="es.startuphero.checkstyle.checks.annotation.ForbiddenAnnotationKeyCheck">
       <property name="annotationForbiddenKeysMap" value="javax.persistence.Column:unique"/>
     </module>
+    <module name="es.startuphero.checkstyle.checks.annotation.RequestBodyAnnotationCheck">
+      <property name="severity" value="info"/>
```

Comment:
`@RequestPart` gibi annotation'lar kullanulabileceğinden dolayı severity'i info olarak belirledim. Eğer genel olarak söz konusu methodlar için strict bir şekilde `@RequestBody` olmasını istiyorsak direkt bu satırı kaldırıp error olarak belirleyebiliriz.